apps/infra/utils/blockchain.py

```
@@ -0,0 +1,255 @@
+
+    Args:
+        address: The address to validate
+
+    Returns:
+        bool: True if address is valid, False otherwise
+    """
+    if not address:
+        return False
+
+    # Check if address has valid format
+    return web3.is_address(address)
```

Ruff (0.11.9)

Undefined name `web3`

(F821)